pkg/apiserver/certificate/certificate.go

```diff
@@ -87,7 +84,7 @@ func ApplyServerCert(selfSignedCert bool, client kubernetes.Interface, aggregato
 			}
 			return true, nil
 		}); err != nil {
-			return nil, fmt.Errorf("error reading TLS certificate and/or key. Please make sure the Secret '%s' is present and has '%s', '%s', and '%s' when selfSignedCert is set to false", TLSSecretName, CACertFile, TLSCertFile, TLSKeyFile)
+			return nil, fmt.Errorf("error reading TLS certificate and/or key. Please make sure the TLS CA (%s), cert (%s), and key (%s) files are provided when selfSignedCert is set to false", CACertFile, TLSCertFile, TLSKeyFile)
```

Comment:
may be useful to have the certDir somewhere in that log message. For example, `CACertFile` is just `"ca.crt"`, not the full path.

Reply:
Done.